scripts/buildtools/install_buildtools.sh

```diff
@@ -70,16 +70,14 @@ function install_go_module {
     # Check for version to go.mod version
     VERSION=$(get_go_version "$1")
 
-    # TODO: When we switch to 1.16 this should be changed to use 'go install'
-    #       instead of 'go get': https://tip.golang.org/doc/go1.16#modules
     if [ -z "$VERSION" ]; then
         echo "Unable to install requested package '$1' (${MODULE}): no version listed in ${SCRIPTPATH}/go.mod"
         exit 1
     else
-        OUTPUT=$(GO111MODULE=on go get "${MODULE}@${VERSION}" 2>&1)
+        OUTPUT=$(GO111MODULE=on go install "${MODULE}@${VERSION}" 2>&1)
```

Comment:
Is `GO111MODULE` needed with `go install`?

Reply:
I think that it used to be required before go 1.12. I'm not sure if its still applies.

Reply:
I think it was auto (built in 1.13+ if go.mod was present) but as of 1.16 it defaults to on. We should probably remove setting it explicitly here.

Reply:
It defaults to on now, but it doesn't hurt to leave it on and might help developers a little behind in keeping their environment up to date. I'd be fine with deleting this in another 6 months or so. It would be nice if `make` reported a warning "Please upgrade your golang to 1.16 or greater".

Reply:
good point, removed all references to GO111MODULE=on

Reply:
To Brian's comment: I think it's okay to set our minimum version for support higher. I believe we call `scripts/check_golang_version.sh` and explicitly tell people to upgrade, if they run configure_dev.sh.

Reply:
It looks like GO111MODULE defaulting to on is linked to GOPATH being deprecated: https://github.com/golang/go/issues/41330

I think it would be a good idea to leave `GO111MODULE=on` until we completely remove `GOPATH`

Reply:
refinement on GOPATH from that golang issue since I was confused and had to read up on it:
$GOPATH/src is going away as a source organization strategy, replaced by modules (and their 99% invisible cache structure inside $GOPATH/pkg). Any build assumption about `go-algorand/` being at `$GOPATH/src/github.com/algorand/go-algorand/` should really really go away (I'm pretty sure it's gone from build but it lingered in integration tests for a while).

$GOPATH/bin is sticking around as the output directory for `go install`

Reply:
Thanks for the clarification Brian. I would guess setting 1.16 in the `go.mod` file would force people to upgrade regardless of `scripts/check_golang_version.sh`, so removing all references to `GO111MODULE=on` should be fine.